apps/arena/lib/arena/game_launcher.ex

```diff
@@ -63,11 +75,60 @@ defmodule Arena.GameLauncher do
     {:noreply, %{state | clients: remaining_clients}}
   end
 
+  def handle_info({:spawn_bot_for_player, bot_client, game_pid}, state) do
+    Finch.build(:get, build_bot_url(game_pid, bot_client))
+    |> Finch.request(Arena.Finch)
+
+    {:noreply, state}
+  end
+
   defp maybe_make_batch_start_at([], _) do
     System.monotonic_time(:millisecond)
   end
 
   defp maybe_make_batch_start_at([_ | _], batch_start_at) do
     batch_start_at
   end
+
+  defp get_bot_clients(missing_clients, game_config) do
+    bot_names = [
```

Comment:
A static list like this should go to a module variable `@bot_names`

Reply:
Changed!